SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,212 @@
+            feedbackTableView.deleteRows(at: [indexPath], with: .fade)
+        }
+        tableView.reloadData()
+    }
+    
+}
+
+// MARK: - TableViewDataSource
+extension ViewController: UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return feedbackData.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: FeedbackTableViewCell.reuseID, for: indexPath) as! FeedbackTableViewCell
+        let feedback = feedbackData[indexPath.row]
+        cell.configure(feedback: feedback)
+        cell.selectionStyle = .none
+        return cell
+    }
+    
+}
+
+// MARK: - DZNEmptyDataSet
+extension ViewController: DZNEmptyDataSetSource, DZNEmptyDataSetDelegate {
+    
+    func customView(forEmptyDataSet scrollView: UIScrollView!) -> UIView! {
+        
+        let customView = CustomUIView(width: view.layer.bounds.width, height: view.layer.bounds.height)
+
+        // Create ParagraphStyle to format label text
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineSpacing = 10
+        
+        // Tell user there is no feedback to display
+        let feedbackLabel = UILabel()
+        feedbackLabel.lineBreakMode = .byWordWrapping
+        feedbackLabel.numberOfLines = 0
+        let title = "No Feedback Yet\n"
+        let subtitle = "See feedback conversations here"
+        let titleAttributes: [NSAttributedString.Key : Any] = [
+            NSAttributedString.Key.font: UIFont._17RobotoMedium!,
+            .paragraphStyle: paragraphStyle,
+            NSAttributedString.Key.foregroundColor: UIColor.titleColor
+        ]
+        let attributedText = NSMutableAttributedString(string: title, attributes: titleAttributes)
+        let subtitleAttributes = [
+            NSAttributedString.Key.font: UIFont._13RobotoRegular!,
+            NSAttributedString.Key.foregroundColor: UIColor.subtitleColor
+        ]
+        attributedText.append(NSAttributedString(string: subtitle, attributes: subtitleAttributes))
+        feedbackLabel.attributedText = attributedText
+        feedbackLabel.sizeToFit()
+        
+        // Start new conversation button
+        let newConversationButton = UIButton()
+        newConversationButton.setTitle("Start Conversation", for: .normal)
+        newConversationButton.backgroundColor = UIColor.themeColor
+        newConversationButton.titleLabel?.font = UIFont._17RobotoMedium
+        newConversationButton.layer.cornerRadius = 22
+        newConversationButton.addTarget(self, action: #selector(handleNewConversationButtonTap), for: .touchUpInside)
+        
+        // Add elements to UIView
+        customView.addSubview(newConversationButton)
+        customView.addSubview(feedbackLabel)
```

Comment:
yea we see this here as well. Try to abstract as much of the similar code as well.